Re-write why_use_GTFS.md and additional recommendation

eliasmbd opened the pull request:
Re-wrote this page to make is consistent with landing page language and logic
- Added more bullets points for easier readability and quicker knowledge memorization by reader. 
- Added links where useful
- Added Continue Learning Section (recommend for other getting started pages)

Here are some screenshots of what it looks like: 

Full page
![127 0 0 1_8000_getting_started_why_use_GTFS_ (2)](https://github.com/MobilityData/elrond.gtfs.org/assets/130911698/0a138974-661c-4571-8f4b-892d89ce021a)

Continue Learning Part
<img width="1139" alt="Screenshot 2024-05-28 at 12 24 03 PM" src="https://github.com/MobilityData/elrond.gtfs.org/assets/130911698/778d1037-3f59-46e6-91fd-462a650ffaff">

eliasmbd committed:
Re-write why_use_GTFS.md

Re-wrote this page to make is consistent with landing page language and logic
- Added more bullets points for easier readability and quicker knowledge memorization by reader. 
- Added links where useful
- Added Continue Learning Section (recommend for other getting started pages)

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
-# Why Use GTFS?
+# Reach more riders and improve public transit with GTFS
+GTFS, the open-source standard for transit data, empowers you to make public transportation more accessible and efficient.
```
I would remove _"the open-source standard for transit data"_ since we're explaining this in the previous page

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
-- **Flexible**: It also offers the possibility of plugging in additional features beyond schedules, like fare information, flexible services, and accessibility information.
-- **Open Source**: by being freely available, data can be shared easily, and developers can create tools, shaping the future of GTFS to better suit their needs.
-- **Community Driven**: developed collaboratively, GTFS ensures the format meets the evolving needs of riders and agencies.
+## GTFS: More Than Just Trip Planning
```
I think this is a bit confusing since all the features we're listing below are actually all about trip planning. Maybe we're confusing this section with the one where we usually highlight other applications of GTFS, for example for research. A more appropiate title to keep the content can be "GTFS: More Than Just Schedules" since we're listing features that go beyond schedules (we can also go back to "GTFS can probably do more than you think")

eliasmbd replied:
Yeah I can stand behind that title

isabelle-dr replied:
+1 for "GTFS can probably do more than you think"

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
 
-The Reference documents for [GTFS Schedule](/documentation/schedule/reference/) and [GTFS Realtime](/documentation/Realtime/reference/) are officially hosted in a [GitHub repository](https://github.com/google/transit), where changes are proposed and reviewed by the community.
 
+    [:octicons-arrow-right-24: Learn more](https://realtimetransit.info/)
```
I'm not sure we want to link to this page, it has good information but it's a bit strange that it just exists on it's own and looks a bit outdated.

eliasmbd replied:
I don't think the information is outdated and is relevant. But, there is a case to be made if this is the best place to put it. It would answer the question of Why GTFS / Realtime though.

isabelle-dr replied:
+1 on notlinking to this site that isn't maintained, and it gives the impression that GTFS Realtime lives on a different site.

I think eventually we want to bring the info that is uniquely appearing on this site here if any (out of scope for this iteration) and archive it. 

IMO this page answers "Why GTFS Realtime" very well.

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
+GTFS empowers riders with accurate schedules, real-time updates, and a wider choice of apps, leading to reduced wait times and informed travel decisions. Here's how GTFS enhances the rider experience:
 
-## Maintained by the community
+- **Up-to-Date Information:** GTFS provides riders with access to real-time schedules, routes, and stop information, facilitating informed travel decisions.
```
Maybe we can change this line to make it less Realtime-specific (the second point already covers that)? removing "real-time" might be enough to do the trick.

eliasmbd replied:
I think so too. Plus having real-time schedules makes it sound weird .

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
+GTFS thrives on community collaboration, ensuring its continuous evolution and relevance:
+
+- **Open Collaboration:** GTFS operates as an open data standard, allowing anyone to contribute changes or improvements.
+- **Guided Evolution:** The specification evolution is overseen by an independent nonprofit organization called [MobilityData](https://mobilitydata.org/), and its changes are guided by the principles of ease of use, backwards-compatibility, and pragmatic change. The specification is primarily concerned with passenger information.
```
the last phrase in this paragraph is relevant but feels a bit out of place here, maybe we can move it upwards as part of the main description for this section?

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
+- **Guided Evolution:** The specification evolution is overseen by an independent nonprofit organization called [MobilityData](https://mobilitydata.org/), and its changes are guided by the principles of ease of use, backwards-compatibility, and pragmatic change. The specification is primarily concerned with passenger information.
+- **Community-driven Development:** The GTFS specification undergoes continuous refinement through input and review from the community, facilitated on platforms like [GitHub](https://github.com/) and [Slack](https://share.mobilitydata.org/slack).
+
+<div class="grid cards" markdown>
```
The cards render a bit off when displaying the window fullscreen on a 13-in macbook, do you know if theres any trick to harmonize the width of the cards? the only workaround that I found was to write longer descriptions so that they max out their width, probably not ideal in this case.

<img width="1409" alt="Screenshot 2024-05-28 at 5 01 12 PM" src="https://github.com/MobilityData/elrond.gtfs.org/assets/104692200/2d05176d-c1b2-435d-a6a6-944edf4ce14a">

eliasmbd replied:
We would have to investigate. Maybe having invisible characters like spacebar would help?

isabelle-dr replied:
@eliasmbd I've tried for another page and it didn't work 😕

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
+## Continue Learning
+<div class="grid cards" markdown>
+
+- :material-skip-previous: [Go to Previous Section: What is GTFS?](../what_is_GTFS)
```
While I like this navigation feature I think we need to be consistent and implement this ideally across the whole site, which might be tricky because not all sections have a natural flow. If folks have a strong opinion in favour of implementing it only in getting started i'd be open to it.

eliasmbd replied:
I would only implement it if we did this across Getting Started

eliasmbd replied:
@isabelle-dr Thoughts on this section and having it in every Getting Started page?

eliasmbd replied:
This feature already exists in Materials for MkDocs
![image](https://github.com/MobilityData/elrond.gtfs.org/assets/130911698/7c49ac8d-e846-4e31-af8e-4f3bc333049f)

Sergiodero commented:
Thanks @eliasmbd! I left some in-line comments, overall I don't have strong opinions over one version vs the other but I do see the point of having more targeted information with a simpler structure based on bullet point form. lmk what you think of my comments.

eliasmbd commented:
> Thanks @eliasmbd! I left some in-line comments, overall I don't have strong opinions over one version vs the other but I do see the point of having more targeted information with a simpler structure based on bullet point form. lmk what you think of my comments.

I think your points are valid. There is a bit more information in this version that I think could be beneficial for the intended target audience.

isabelle-dr commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,104 @@
-- The specification is primarily concerned with passenger information
-- Changes to the specification should be backwards-compatible
-- Speculative features are discouraged
+-   :material-bus-alert:{ .lg .middle } __Benefits of Realtime Information (RTI)__
```
remove "RTI" which is not used anywhere on this site

isabelle-dr reviewed (changes requested):
Thank you for working on this @eliasmbd!
Love the formatting changes with bulletpoints, and the call-out about Improved Rider experience. 

Reading this, I felt like I was reading different versions of the same arguments multiple times. 
I think it would be stronger if each bullet point/section was communicating one unique argument. 

Examples:
**Arguments that all say "GTFS is developed publicly by an open community".** 
* Open Collaboration: GTFS operates as an open data standard, allowing anyone to contribute changes or improvements. (in Maintained by the Community section).
* Community-driven Development: The GTFS specification undergoes continuous refinement through input and review from the community, facilitated on platforms like [GitHub](https://github.com/) and [Slack](https://share.mobilitydata.org/slack). (in Maintained by the Community section).
* Collaborative Development: GTFS is collaboratively developed, ensuring the format remains relevant to meet the evolving needs of riders and agencies. (in Simplified Data Sharing section).

**Arguments that all say "publishing open GTFS data means a wide range of apps will reuse the data, leading to riders being able to use an app that best serves them".**
* Diverse Applications: This fosters a broad spectrum of applications and services, ranging from popular trip planners to innovative solutions. (in Open Data: More Opportunity and Choices section).
* Expanded App Reach: Access to GTFS data allows app developers to reach a wider audience of transit riders, enhancing the usability and functionality of transit apps. (in Wider User Base for Apps & Services).
* Wider App Choice: Access to GTFS data enables riders to choose from a variety of transit apps, ensuring they find the app that best suits their needs and preferences. (in the Improved Rider Experience" section).

@eliasmbd you have all the unique arguments listed here, maybe each of them is a section?
 
1. widely used
2. **free** and developed publicly by an open community
3. simple and easy to use
4. GTFS can probably do more than you think
5. improved rider experience
6. access to a wide range of applications

eliasmbd commented:
> Reading this, I felt like I was reading different versions of the same arguments multiple times.

@isabelle-dr I was actually thinking the same and wanted to see if it was only my impression. I do feel there is some similarities in some points. My biggest issues was trying to consolidate the points around data. As for community, there are some simple point allocations to be done.

isabelle-dr commented:
@eliasmbd do you want to have another go at it?

eliasmbd commented:
> @eliasmbd do you want to have another go at it?

Doing that right now ;) Looking up how the grid cards are formatted. They seem to be dependent on how the whole site formats the grids. If that doesn't work, I will replace with buttons instead. 

As for content I will revise it and post another version.

eliasmbd commented:
@Sergiodero @isabelle-dr 

Which do you prefer? Grid Cards or Buttons? 
![image](https://github.com/MobilityData/elrond.gtfs.org/assets/130911698/ae6a0339-5913-4789-b7a8-2e7c33250edc)

Sergiodero commented:
> Which do you prefer? Grid Cards or Buttons?

I think for this case I prefer buttons, but we might have to revise this in other sections to be consistent (e.g. in Technical Documentation and Community)

eliasmbd commented:
> > Which do you prefer? Grid Cards or Buttons?
> 
> I think for this case I prefer buttons, but we might have to revise this in other sections to be consistent (e.g. in Technical Documentation and Community)

I think there is a valid use case for both types. But in this case I agree with you.

eliasmbd committed:
Update why_use_GTFS.md

eliasmbd commented:
Applied many of the changes as asked above. 

The webpage looks like this: 
![127 0 0 1_8000_getting_started_why_use_GTFS_ (3)](https://github.com/MobilityData/elrond.gtfs.org/assets/130911698/4913ea4e-6c25-43ae-915f-937172bb2977)

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,51 @@
+## GTFS can probably do more than you think
 
-Additionally, GTFS data is now being used by a variety of software applications including data visualization and analysis tools for planning and research. Having up-to-date and high quality data provides accurate transit information not just to riders, but also to planners and policymakers who are able to better understand how transit is being used in their communities. 
+While trip planning is a core function, GTFS offers additional features that enhance the transit experience:
```
I suggest a few tweaks in this text, replacing it with the following: _"While schedule information is at its core, GTFS also offers additional features that enhance the transit experience, such as:"_

Sergiodero commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,42 +1,51 @@
+## Improved Rider Experience
 
-## Maintained by the community
+GTFS empowers riders with accurate schedules, real-time updates, and a wider choice of apps, leading to reduced wait times and informed travel decisions. Here's how GTFS enhances the rider experience:
```
I'm not too sure about this, but I think that maybe we can remove the segment that aludes to a wider choice of apps, since it's being covered in the next section.

Sergiodero commented:
Thanks @eliasmbd I think the content is in much better shape now! I left a couple of in-line comments but overall I like it and my only (small) concern is that the Simple and Easy to Use section ended up a bit lighter than other sections, but at this point I can't think of something to add. Again this is something minor and if you're okay with it we can leave it as is.

I'll let @isabelle-dr take a look as well and share her thoughts before approving.

eliasmbd committed:
Update why_use_GTFS.md

Smaller editorial changes as commented by @Sergiodero

eliasmbd commented:
> Thanks @eliasmbd I think the content is in much better shape now! I left a couple of in-line comments but overall I like it and my only (small) concern is that the Simple and Easy to Use section ended up a bit lighter than other sections, but at this point I can't think of something to add. Again this is something minor and if you're okay with it we can leave it as is.
> 
> I'll let @isabelle-dr take a look as well and share her thoughts before approving.

I applied those suggestions. As of weight of sections, we can either find another heavy argument or leave it as is. Some single arguments might be heavier despite only having a single bulletpoint.

isabelle-dr reviewed (approved):
LGTM, thank you for making the changes @eliasmbd!

If you wanted a second bulletpoint in _Simple and Easy to Use_, you could have (taken from the [guiding principles](https://gtfs.org/schedule/process/#guiding-principles)):
- Backwards compatible: when updating the specification, existing feeds remain valid and maintain compatibility with existing parsers.

eliasmbd committed:
Update why_use_GTFS.md

Added: Backwards compatible: when updating the specification, existing feeds remain valid and maintain compatibility with existing parsers.
